pkg/chain/ethereum/bonded_ecdsa_keep_handle.go

```diff
@@ -0,0 +1,343 @@
+package ethereum
+
+import (
+	"fmt"
+	"sort"
+	"time"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/keep-network/keep-common/pkg/chain/ethereum/ethutil"
+	"github.com/keep-network/keep-common/pkg/subscription"
+	"github.com/keep-network/keep-ecdsa/pkg/chain"
+	"github.com/keep-network/keep-ecdsa/pkg/chain/gen/contract"
+	"github.com/keep-network/keep-ecdsa/pkg/ecdsa"
+	"github.com/keep-network/keep-ecdsa/pkg/utils/byteutils"
+)
+
+type bondedEcdsaKeep struct {
+	handle   *ethereumChain
+	contract *contract.BondedECDSAKeep
+
+	contractAddress common.Address // FIXME Expose address on contract.BondedECDSAKeep?
```

Comment:
> // FIXME Expose address on contract.BondedECDSAKeep?

Not sure if I follow - do we want to create multiple instances of `*contract.BondedECDSAKeep`?

Reply:
The open question is whether we should be exposing this address via an exported function. This note predates `chain.KeepID` and the `ID()` function (i.e., the answer was “yes” haha).